feature-libs/cart/base/root/models/cart.model.ts

```diff
@@ -118,6 +118,7 @@ export interface CartItemComponentOptions {
   displayAddToCart?: boolean;
   addToCartString?: string;
   cartType?: CartType;
+  disableItemLink?: boolean;
```

Comment:
same as above
not considered as breaking change as added property is optional and DOM structure stayed unchanged for existing customers.